custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -822,7 +822,13 @@
         "datatype": "date",
         "type": "raw",
         "property_name": "last_reported_fever_date"
-      }
+      },
+      {
+        "column_id": "modified_on",
+        "datatype": "datetime",
+        "type": "raw",
+        "property_name": "modified_on"
+      },
```

Comment:
actually is this trailing comma not going to crash?

Reply:
oh yeah it will, I removed that but missed while pushing